### PR TITLE
html & api fetcher service framework, and test page created

### PR DESCRIPTION
**Pull request by cabe7**
Screenshot from 2019-08-27 16-54-20

Shown output if from the HTML view (i.e. on page load), pressing B1 triggers api for refresh of shown results

### CODE REVIEW

**Comment by cabe7** [comment]
Pushed to establish a basic presence on these endpoints. I am continuing to work on this branch and will be submitting further pushes.

File: app/controllers/activities_controller.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
+# BROWSER activities controller - not api
 class ActivitiesController < ApplicationController
-
-skip_before_action :authenticate_user!, only: :index
-
+  skip_before_action :authenticate_user!, only: :index
   def index
     # e.g. localhost:3000/plans/1/activities
     @activities = policy_scope(Activity)
+    fetcher = ActivitiesFetcherService.new
```
Comment (maxwrf) [inline app/controllers/activities_controller.rb]: isnt the application controller called before every page?

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -1 +1,22 @@
...
+}
+
+const activities = document.getElementById("activities");
+const b1 = document.getElementById("b1");
+
+b1.addEventListener("click", (event) => {
+  fetch("api/v1/activities")
+  .then(response => response.json())
+  .then((data) => {
+    console.log(data);
```
Comment (maxwrf) [inline app/javascript/packs/application.js]: remove?

File: app/services/activities_fetcher_service.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class ActivitiesFetcherService
+  def initialize
+    @test = 1
+  end
+
+  def fetch(results)
+    generate_test_activity(results)
+  end
+
+  def generate_test_activity(num)
```
Comment (maxwrf) [inline app/services/activities_fetcher_service.rb]: this where we are actually gonna fetch data from google/tripadvisor?

**Pull request merged**